themes/my-test-theme-dark/README.md

```diff
@@ -0,0 +1,12 @@
+# KubeColor Themes
```

Comment:
Suggest moving this file to one dir up: `themes/README.md`

And then have the `themes/my-test-theme-dark/README.md` only contain the info you request

Reply:
I think we don't need a readme there, only comments in the theme file ?

Reply:
Yep you're right. Even simpler